LWA/LWA_bifrost.py

```diff
@@ -815,12 +815,15 @@ def __init__(
         self.ntime_gulp = ntime_gulp
         self.accumulation_time = accumulation_time
 
-        self.antennas = antennas
+        self.station = station
         locations = numpy.empty(shape=(0, 3))
-        for ant in self.antennas:
+        for ant in self.station.antennas:
             locations = numpy.vstack((locations, [ant.stand[0], ant.stand[1], ant.stand[2]]))
         locations = numpy.delete(locations, list(range(0, locations.shape[0], 2)), axis=0)
```

Comment:
I think this line could easily just slice every other starting at the 1th entry, since we're deleting every other starting at 0. 
```suggestion
        locations = locations[1::2]
```

Reply:
also in favor of just making the for loop above only iterate like
```
for ant in self.station.antennas[1::2]:
do things
```

Reply:
You could even:
```
locations = numpy.array([(ant.stand.x, ant.stand.y, ant.stand.z) for ant in self.station.antennas[::2]])
```

Reply:
so much prettier